egs/libri_css/s5_mono/local/dscore.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# Copyright  2020   Desh Raj
+
+# Apache 2.0.
+#
+# This script installs a fork of the dscore toolkit 
+# (https://github.com/nryant/dscore), which also supports
+# evaluating the overlapping regions only. It then scores
+# the output sys_rttm based on the provided ref_rttm.
+
+score_overlaps_only=true
+
+echo "$0 $@"  # Print the command line for logging
+
+set -e
+
+. ./path.sh
+. parse_options.sh || exit 1;
+
+if [ $# != 2 ]; then
+  echo "Usage: $0 <ref-rttm> <hyp-rttm>"
+  echo "e.g.: $0 data/test/rttm exp/test_diarization/rttm"
+  exit 1;
+fi
+
+ref_rttm=$1
+hyp_rttm=$2
+  
+if ! [ -d dscore ]; then
+  git clone https://github.com/desh2608/dscore.git -b libricss --single-branch
+  cd dscore
+  python3 -m pip install --user -r requirements.txt
+  cd ..
+fi
+
+# Create per condition ref and hyp RTTM files for scoring per condition
+mkdir -p tmp
+conditions="0L 0S OV10 OV20 OV30 OV40"
+cp $ref_rttm tmp/ref.all
+cp $hyp_rttm tmp/hyp.all
+for rttm in ref hyp; do
+  for cond in $conditions; do
+    cat tmp/$rttm.all | grep $cond > tmp/$rttm.$cond
+  done
+done
+
+echo "Scoring all regions..."
+for cond in $conditions 'all'; do
+  echo -n "Condition: $cond: "
+  ref_rttm_path=$(readlink -f tmp/ref.$cond)
+  hyp_rttm_path=$(readlink -f tmp/hyp.$cond)
+  cd dscore && python3 score.py -r $ref_rttm_path -s $hyp_rttm_path --global_only && cd ..
```

Comment:
Since you `set -e`, the `&&` are useless; if any command in this chain fails, the script exists with an error. It would be more readable if written in a normal sequence of commands, like
```suggestion
  cd dscore
  python3 score.py -r $ref_rttm_path -s $hyp_rttm_path --global_only
  cd ..
```
Same at line 62.